src/components/modules/Authentcation/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,5 +1,86 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { AUTHURL } from "../../../../constans/EndPoints/ENDPOINT";
+import { toast } from "react-toastify";
+import { VALIDATIONS } from "../../../../constans/Validation/Validation";
+import { Forgetpassword } from "../../../../../interface/Interface";
 
 export default function ForgetPassword() {
-  return <div>ForgetPassword</div>;
+  const navigate = useNavigate();
+  const navigateLogin = () => {
+    navigate("/Login");
+  };
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Forgetpassword>();
+  const onsubmit = async (data: Forgetpassword) => {
+    try {
+      const response = await axios.post(AUTHURL.FORGET, data);
+      console.log(response.data);
+      toast.success(response.data.message);
+
+      navigate("/reset-password");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <>
+      <h1 className="py-5 font-bold text-3xl text-auttextcolor">
+        Forgot password
+      </h1>
+      <div className="p-10 w-full flex justify-start gap-8">
+        <div
+          className="bg-authchoosuserbackgroundcolor w-48 text-center py-5 rounded-xl transition-all hover:outline cursor-pointer  outline-auttextcolor "
+          onClick={navigateLogin}

```

Comment:
We shouldn't be giving onClick event to divs as divs are not interactive elements which means that they will not be accessibile to users using keyboard to navigate the website.

In our case, this should be a link as it's main purpose is navigation. We can use the Link component from react-router-dom to achieve this.